modules/global-classes/global-classes-repository.php

```diff
@@ -45,15 +47,18 @@ public function put( string $id, array $value ) {
 		unset( $value['id'] );
 
 		if ( ! isset( $all->get_items()[ $id ] ) ) {
-			throw new \Exception( "Global class with id ${id} not found" );
+			throw new \Exception( "Global class with id {$id} not found" );
 		}
 
 		if ( $value === $all->get_items()[ $id ] ) {
 			return $value;
 		}
 
+		[, $parsed] = Style_Parser::make( Style_Schema::get() )
+			->parse( $value );
```

Comment:
it validate the values twice, once in the `validate_callback` and once here, maybe we should use `sanitize_callback` instead of this code?